diff --git a/node_modules/pdfjs-dist/build/pdf.mjs b/node_modules/pdfjs-dist/build/pdf.mjs
index 7906d1a..c8a2116 100644
--- a/node_modules/pdfjs-dist/build/pdf.mjs
+++ b/node_modules/pdfjs-dist/build/pdf.mjs
@@ -5443,6 +5443,14 @@ class FontFaceObject {
 
 ;// CONCATENATED MODULE: ./src/display/node_utils.js
 
+Promise.withResolvers = function () {
+  let resolve, reject;
+  const promise = new Promise((res, rej) => {
+    resolve = res;
+    reject = rej;
+  });
+  return { promise, resolve, reject };
+};
 
 if (isNodeJS) {
   var packageCapability = Promise.withResolvers();
